abbyy_to_epub3/create_epub.py

```diff
@@ -144,8 +144,8 @@ def __init__(self, item_dir, item_identifier, item_bookpath, debug=False, args=F
 
         super(Ebook, self).__init__(item_dir, item_identifier, item_bookpath)
 
-        self.tmpdir = tempfile.TemporaryDirectory()
-        self.cover_img = '{}/cover.png'.format(self.tmpdir.name)
+        self.tmpdir = tempfile.TemporaryDirectory(dir=mkdir_p(tmpdir))
+        self.cover_img = '{}/cover.bmp'.format(self.tmpdir.name)
```

Comment:
Note that this needed change really should be in a separate feature implementation. But okaying it for expediency.